web/src/styles/app/layout.css

```diff
@@ -53,3 +53,8 @@ footer {
 .w-3 {
   width: 3rem;
 }
+
+.sticky-top {
+  position: sticky;
```

Comment:
If we need to support IE, we can't rely on `position: sticky;`.  I think that's a product question.

@jeromeleecms @NAretakis - are there specific minimum versions of browsers that we should be targeting?  Should we informally poll states to ask what browsers and versions they use?  (Have we already and I forgot? 😛)

Reply:
i see this particular feature as a progressive enhancement -- if your browser supports it, you get a nice sticky header; if not, the experience is still perfectly fine but you now have to scroll a little if you forget which activity you're working on

we should def ask states what browser and version they're using (i guess we can use google analytics too and see for ourselves)

Reply:
I agree with Brendan. I can do some informal polling on browsers, but as long as they can go through the experience in some way, we should be ok.